internal/cmd/cmd_get.go

```diff
@@ -38,22 +38,26 @@ func init() {
 	CommandRegistry.AddCommand(cGET)
 }
 
-func newGETRes(obj *object.Obj) *CmdRes {
+func newGETRes(obj *object.Obj) (*CmdRes, error) {
```

Comment:
We do not want to return `error` from this function. Create a CmdRes object instead.